README.md

```diff
@@ -102,6 +109,16 @@ Or as `GeoDataFrame`:
 occurrences('813ca36b-91e3-4a18-b408-60b27a1942ef', format='geodf')
 ```
 
+#### `Occurrences` parameters
+
+* `id_state`: required, the state id;
+* `id_cities`: optional, the city id or a list of city ids;
+* `type_occurrence`: optional, the type of occurrence. Possible values are `"all"`, `"withVictim"`, `"withoutVictim"`; 
+* `initial_date`: optional, the initial date of the occurrences. The date format are `YYYY-MM-DD`, `YYYY/MM/DD` and `YYYYMMDD`;
+* `final_date`: optional, the final date of the occurrences. The date format are `YYYY-MM-DD`, `YYYY/MM/DD` and `YYYYMMDD`;
+* `max_parallel_requests`: optional, the maximum number of parallel requests to the API; Default: 16;
+* `format`: optional, the format of the result, `"dict"` for dictionary, `"df"` for `DataFrame` or `"geodf"` for `GeoDataFrame`;
```

Comment:
O que você acha de formatar essas sessões de forma mais estruturada, com uma tabela?

Por exemplo:

| Name | Required | Description | Type | Default value | Example |
|---|---|---|---|---|---|
| `id_state` | ✅ | ID of the state | string | `None` | `'813ca36b-91e3-4a18-b408-60b27a1942ef'` |

Reply:
Massa! Vou adotar, sim.
Acho qeu visualmente chama mais atenção.